baselines/common/tests/envs/mnist_env.py

```diff
@@ -14,6 +13,7 @@ def __init__(
             episode_len=None,
             no_images=None
     ):
+        import filelock
```

Comment:
any reason for the late imports here?

Reply:
so that we don't need filelock module for the tests that don't use MnistEnv - not a really big reason, but sometimes comes handy.